Source/MQTTnet/Server/Internal/MqttClientSubscriptionsManager.cs

```diff
@@ -31,7 +31,8 @@ public sealed class MqttClientSubscriptionsManager : IDisposable
         readonly Dictionary<string, MqttSubscription> _subscriptions = new Dictionary<string, MqttSubscription>();
 
         // Use subscription lock to maintain consistency across subscriptions and topic hash dictionaries
-        readonly AsyncLock _subscriptionsLock = new AsyncLock();
+        //readonly SemaphoreSlim _subscriptionsLock = new SemaphoreSlim(1);
```

Comment:
Please remove commented out code if not required any longer.

Reply:
removed, thanks